README.md

```diff
@@ -95,7 +95,14 @@ particular namespace.
 The easiest way to get started is to use the `docker-compose.yaml` in the `dev`
 folder.
 
-First, generate the needed certificates:
+First, install the tools needed:
+
+    go get -u go.aporeto.io/tg
+    go get -u go.aporeto.io/elemental/cmd/elegen
+    go get -u go.aporeto.io/regolithe/cmd/rego
+    go get -u github.com/go-bindata/go-bindata
```

Comment:
Hi 

Thanks for the contribution!

The recommended way to install go binaries is now with `go install`:

```suggestion
    go install go.aporeto.io/tg@master
    go install go.aporeto.io/elemental/cmd/elegen@master
    go install go.aporeto.io/regolithe/cmd/rego@master
    go install github.com/aporeto-inc/go-bindata@master
```

Notice the change for go-bindata. It is uses our fork that fixes multiple formatting error. Something shady happened with the official go-bindata a few years ago.